Source/Core/DiscIO/NANDContentLoader.cpp

```diff
@@ -248,26 +248,6 @@ void CNANDContentManager::ClearCache()
   m_map.clear();
 }
 
-bool AddTicket(const IOS::ES::TicketReader& signed_ticket)
-{
-  if (!signed_ticket.IsValid())
```

Comment:
Yes. ImportTicket already checks for validity, so keeping it would be redundant.